tcms/testcases/tests/test_views.py

```diff
@@ -179,6 +179,21 @@ def setUpTestData(cls):
         user_should_have_perm(cls.tester, 'testcases.add_testcase')
         user_should_have_perm(cls.tester, 'testcases.view_testcase')
 
+    def test_form_invalid(self):
```

Comment:
There also doesn't seem to be a descendant of PermissionsTestCase for this view.